biogeochem/EDPhysiologyMod.F90

```diff
@@ -2928,7 +2943,9 @@ subroutine CWDInput( currentSite, currentPatch, litt, bc_in)
        do dcmpy=1,ndcmpy
           dcmpy_frac = GetDecompyFrac(pft,leaf_organ,dcmpy)
           litt%leaf_fines_in(dcmpy) = litt%leaf_fines_in(dcmpy) + &
-               (leaf_m_turnover+repro_m_turnover) * plant_dens * dcmpy_frac
+               (leaf_m_turnover+repro_m_turnover + &
+               leaf_herbivory * herbivory_element_use_efficiency) * &
```

Comment:
Just to help me understand the steps, will the leaf litter here already take into account the leaf mass that is consumed and "stays" in the animal as food, and only a portion returning to the litter as "manure"?  Is that the herbivory_element_use_efficiency parameter, or something else?  Because I'm also confused about what portion of the litter is food, is manure, and/or to the atmosphere, and if all 3 paths are tracked?